tests/fixtures/amsmath.md

```diff
@@ -181,13 +190,32 @@ equation environment, in list:
 <li>
 <div class="math amsmath">
 \begin{equation}
-  a = 1
-  \end{equation}
+a = 1
+\end{equation}
 </div>
 </li>
 </ul>
 .
 
+equation environment, in block quote:
+.
+> \begin{matrix}
+> -0.707 &  0.408 &  0.577 \\
+> -0.707 & -0.408 & -0.577 \\
+> -0.    & -0.816 &  0.577
+> \end{matrix}
+.
+<blockquote>
+<div class="math amsmath">
+\begin{matrix}
+-0.707 &amp;  0.408 &amp;  0.577 \\
+-0.707 &amp; -0.408 &amp; -0.577 \\
+-0.    &amp; -0.816 &amp;  0.577
+\end{matrix}
+</div>
+</blockquote>
```

Comment:
I compatibility with Jupyter is of importance, then the HTML should be different here:

```html
<blockquote>
<div class="math amsmath">
\begin{matrix}
&gt; -0.707 &amp;  0.408 &amp;  0.577 \\
&gt; -0.707 &amp; -0.408 &amp; -0.577 \\
&gt; -0.    &amp; -0.816 &amp;  0.577
&gt; \end{matrix}
</div>
</blockquote>
```

This is how it is rendered in Jupyter Lab:

![jupyterlab](https://github.com/user-attachments/assets/00056da7-5c60-43f5-ba14-b59cd74d2d7f)

Or should this be treated as a Jupyter issue? In Jupyter, one would write the markdown as:

```markdown
> \begin{matrix}
-0.707 &  0.408 &  0.577 \\
-0.707 & -0.408 & -0.577 \\
-0.    & -0.816 &  0.577
\end{matrix}
```

Given that Jupyter is so wide spread, it's seems prudent to also include this example in the tests.

Reply:
> Or should this be treated as a Jupyter issue?

very much so, that is definitely very wrong 😅

I'm interested in "correct parsing", not mimicking bugs of other programs 😬

> If compatibility with Jupyter is of importance

As you note, what you are actually talking about here is only the jupyter lab implementation, 
which is relies on things like markedjs and mathjax,
and is known to have a bunch of issues with math rendering (https://github.com/jupyterlab/jupyterlab/issues/8645, https://github.com/jupyterlab/jupyterlab/issues/14570, etc)

by contrast, try this in https://code.visualstudio.com/docs/datascience/jupyter-notebooks and you will get perfectly correct math 😄

<img width="783" alt="image" src="https://github.com/user-attachments/assets/08df06b1-35b3-48c8-95a5-15f2ca7019ff">

Reply:
> it seems prudent to also include this example in the tests

I've added it for completeness, but to be clear, this is just a case of block quote lazy continuation: https://spec.commonmark.org/0.30/#example-233

Reply:
I've opened a Jupyter issue: https://github.com/jupyterlab/jupyterlab/issues/16755